### PR TITLE
added quotes by Barack Obama, Mae West & Robert Frost

### PR DESCRIPTION
**Pull request by izzyben20**
# Pull Request

I added quotes by Barack Obama, West and Robert Frost

## Related Issue:

Closes #XXXXX

### CODE REVIEW

**Review by mertoenjosh — DISMISSED** [review]
Seen ✔️✔️

File: src/config/QuoteList.ts [hunk 1]
```diff
@@ -97,4 +97,20 @@ export const QuoteList: Quote[] = [
...
+  {
+    text: "If you're walking down the right path and you're willing to keep walking, eventually you'll make progress.",
+    author: "Barack Obama",
+  },
+  {
+    text: "Where you are right now doesn't have to determine where you'll end up.",
+    author: "Barack Obama",
+  },
+  {
+    text: "You only live once,but if you do it right, once is enough",
```
Comment (nhcarrigan) [inline src/config/QuoteList.ts:109]:
```suggestion
    text: "You only live once, but if you do it right, once is enough",
```
Just a quick spacing issue.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**